frontend-ssr/pages/submissions.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Head from 'next/head';
+import Footer from '~/components/footer/Footer.component';
+import Header from '~/components/header/Header.component';
+
+import styles from '../styles/pages/default-page.module.scss';
```

Comment:
Why are we importing those styles?

Also as a general rule, let's always add the styles import last, just like in the other files, separated by a newline from the other import. It's easier to see what styles are we getting.